tests/fixtures/testModel.ts

```diff
@@ -209,6 +209,324 @@ export interface TestModel extends Model {
   extensionComment?: boolean;
 
   keyofLiteral?: keyof Items;
+
+  namespaces?: {
+    simple: NamespaceType;
+    inNamespace1: Namespace1.NamespaceType;
+    typeHolder1: Namespace1.TypeHolder;
+    inModule: Namespace2.Namespace2.NamespaceType;
+    typeHolder2: Namespace2.TypeHolder;
+  };
+
+  defaults?: {
+    basic: DefaultsClass;
+    replacedTypes: ReplaceTypes<DefaultsClass, boolean, string>;
+    /**
+     * @default undefined
+     */
+    defaultUndefined?: string;
+    /**
+     * @default null
+     */
+    defaultNull: string | null;
+    /**
+     * @default
+     * {
+     *   "a": "a",
+     *   "b": 2
+     * }
+     */
+    defaultObject: { a: string; b: number };
+    /**
+     * @default `\`"'\"\'\n\t\r\b\f\v\0\g\x\\`//\0, \v is not supported...
+     *
+     */
+    stringEscapeCharacters: undefined; //type is not really interesting
+    /**
+     * @default //Comment1
+     * 4
+     * //Comment2
+     *
+     */
+    comments: undefined; //type is not really interesting
+    /**
+     * @default {
+     * //Alma
+     * `\\`: '\n'
+     *
+     * }
+     *
+     */
+    jsonCharacters: undefined; //type is not really interesting
+  };
+
+  jsDocTypeNames?: {
```

Comment:
This types are different because of their jsdoc comments. Should generate different TSOA types for it (with, of course, different names).
Test checks situations which were handled previously as Partial_any_ to cover a more special situation